Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/tools/TextToolIntegrationTest.java

```diff
@@ -379,6 +397,25 @@ public void testCheckBoxSizeAndContentAfterFormattingToDubaiAndStc() {
 		}
 	}
 
+	@Test
+	public void testInputTextAndFormatByTextSize() {
+		enterTestText();
+
+		ArrayList<FormattingOptions> sizes = new ArrayList<>();
+		sizes.add(FormattingOptions.SIZE_30);
+		sizes.add(FormattingOptions.SIZE_40);
+		sizes.add(FormattingOptions.SIZE_60);
+
+		for (FormattingOptions size : sizes) {
```

Comment:
Please avoid for loops in tests, they make it hard to figure out what exactly failed afterwards.